scrabble/scrabble.py

```diff
@@ -32,14 +32,32 @@
 
 def loadWords():
     '''
-    Loads the words from the file into (wordsLoaded)
+    Loads the words from the file into 'wordsLoaded'
+    A Dictionary seperating each letter of the english alphabet
     '''
+    allWords = {'a': [], 'b': [], 'c': [], 'd': [], 'e': [],
+                'f': [], 'g': [], 'h': [], 'i': [], 'j': [],
+                'k': [], 'l': [], 'm': [], 'n': [], 'o': [],
+                'p': [], 'q': [], 'r': [], 's': [], 't': [],
+                'u': [], 'v': [], 'w': [], 'x': [], 'y': [],
+                'z': []}
+
     inFile = open(WORD_FILE, 'r')
-    wordsLoaded = []
-    for line in inFile:
-        wordsLoaded.append(line.strip().lower())
-    print(len(wordsLoaded), "words loaded.")
-    return wordsLoaded
+    wordList = []
+    for e in inFile:
+        wordList.append(e.strip().lower())
+    inFile.close()
+
+    counter = 0
+    for letter in allWords:

```

Comment:
This isn't a letter, it is a `word`.

Reply:
yeah its the key

Reply:
btw, in python u can do `for key, value in dict:`. This way you don't need to do `allWords[letter]` below (you would just do `value.append...`.

Reply:
Oh right! I completely forgot about that, I'll keep it in mind